src/expression/nn.cpp

```diff
@@ -17,6 +19,79 @@ dw::Linear::Linear(int units, std::string name)
     m_info.impl().attrs["units"] = units;
 }
 
+template<class T>
+struct get_in {
+    static T get(const std::vector<const dw::Attribute*>& args, int idx) {
```

Comment:
I believe there is no sense to create vector, possible to get attributes directly from map based on order vector

Reply:
done